packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -226,6 +237,67 @@ export const PaymentForm = ({
       </>
     );
 
+  const buttons = onCancel ? (
+    <div className="button-row">
+      <Localized id="payment-cancel-btn">
+        <button
+          data-testid="cancel"
+          className="settings-button cancel secondary-button"
+          onClick={onCancel}
+        >
+          Cancel
+        </button>
+      </Localized>
+
+      <Localized
+        id={submitButtonL10nId ? submitButtonL10nId : 'payment-update-btn'}
+      >
+        <SubmitButton
+          data-testid="submit"
+          className="settings-button primary-button"
+          name="submit"
+          disabled={inProgress}
+        >
+          {inProgress ? (
+            <span data-testid="spinner-update" className="spinner">
+              &nbsp;
+            </span>
+          ) : submitButtonCopy ? (
+            <span>{submitButtonCopy}</span>
+          ) : (
+            <span>Update</span>
+          )}
+        </SubmitButton>
+      </Localized>
+    </div>
+  ) : submit ? (
+    <div className="button-row">
+      <Localized id="payment-submit-btn">
+        <SubmitButton
+          data-testid="submit"
+          name="submit"
+          disabled={!allowSubmit}
+        >
+          {showProgressSpinner ? (
+            <span data-testid="spinner-submit" className="spinner">
+              &nbsp;
+            </span>
+          ) : (
+            <Localized
+              id={
+                submitButtonL10nId
+                  ? submitButtonL10nId
+                  : payButtonL10nId(customer!)
```

Comment:
We definitely don't have a `customer` in a couple of flows.  Instead of asserting `customer`, I think we should update `payButtonL10nId` to accept null or undefined.

Reply:
This is an existing method, I think it would be a good thing to clean up also, can you file an issue and tag it for a maintenance sprint?

Reply:
I mean we are just removing a `!` here at the call site and then change the param type of `payButtonL10Id` to `(c?: Customer | null)`.  Filing an issue, triaging, creating another PR, having someone review it seems like a lot of unnecessary effort.  Would you accept a patch that you can apply? 

In this case, TypeScript is correct about the type of `customer` here.